rules/windows/malware/win_mal_ryuk.yml

```diff
@@ -11,10 +11,14 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\net.exe stop "samss" *'
-            - '*\net.exe stop "audioendpointbuilder" *'
-            - '*\net.exe stop "unistoresvc_?????" *'
+        ProcessName|contains:
+            - 'net.exe'
```

Comment:
I would rather use:

```
        ProcessName|endswith:
            - '\net.exe'
            - '\net1.exe'
```

That's a normal way to detect `net` execution %)

Reply:
Fixed :)